.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+# Description
+
+Please summarise the changes.
+
+# Related issues
+
+Please mention any github issues addressed by this PR.
+
+# Checklist
+
+- [ ] I have performed a self-review of my code.
+- [ ] I have commented hard-to-understand parts of my code.
+- [ ] I have made corresponding changes to the public API documentation.
+- [ ] I have added tests that prove my fix is effective or that my feature works.
+- [ ] I have updated the changelog with any user-facing changes.
```

Comment:
Should PRs bump the extension_version in `metadata.py`?

Reply:
We generally do that only on the release branch just before making a release.